source/agora/common/Config.d

```diff
@@ -193,6 +188,9 @@ public struct ValidatorConfig
 
     // If the enrollments will be renewed or not at the end of the cycle
     public bool recurring_enrollment = true;
+
+    /// Used only by the testing framework to override the timestamp for genesis block
```

Comment:
this is very far from ideal to put this data here: what I would need is a way to be able to get the start time of the test in the nominator

I can fix this in a follow up PR where I change getExpectedBlockTime from private to protected and override it in the TestNominator

Reply:
I made the improvement as part of this PR instead of raising a separate one, hopefully this will be enough to merge this PR